examples/ProjectExample/pipinstaller/install_Functions.py

```diff
@@ -43,9 +41,7 @@ def checkPackage(pkg):
 #Function to install pip
 #######################################
 def installPip(printOut = False):
-    pipPath = resourcesPath / "get-pip.py"
-    print(pipPath)
-    cmd = f"py -3 {pipPath} --user" if sys.platform == "win32" else f"python3 {pipPath} --user"
```

Comment:
Wait, what was wrong here?

Reply:
might want to add command though instead. f"{cmd} {pipPath}" if this is still valid